### PR TITLE
Use aave as a pricing asset

### PR DESCRIPTION
**Pull request by gtaschuk**
For historical pricing data on the WSB ETP

### CODE REVIEW

File: src/mappings/helpers/constants.ts [hunk 1]
```diff
@@ -50,6 +50,16 @@ let wbtcAddressByNetwork: AddressByNetwork = {
...
 
+let aaveAddressByNetwork: AddressByNetwork = {
+  mainnet: '0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9',
+  kovan: '0xB597cd8D3217ea6477232F9217fa70837ff667Af',
+  goerli: '0x0000000000000000000000000000000000000000',
+  rinkeby: '0xBf6f1a1C0D494313d0053BDd97714330b6FdAff3',
+  polygon: '0xD6DF932A45C0f255f85145f286eA0b292B21C90B',
+  arbitrum: '0x0000000000000000000000000000000000000000',
+  dev: '0x0000000000000000000000000000000000000000',
+}
```
Comment (EndymionJkb) [inline src/mappings/helpers/constants.ts:61]:
```suggestion
};
```
Lint

File: src/mappings/helpers/constants.ts [hunk 2]
```diff
@@ -50,6 +50,16 @@ let wbtcAddressByNetwork: AddressByNetwork = {
   dev: '0xcD80986f08d776CE41698c47f705CDc99dDBfB0A',
 };
 
+let aaveAddressByNetwork: AddressByNetwork = {
+  mainnet: '0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9',
+  kovan: '0xB597cd8D3217ea6477232F9217fa70837ff667Af',
+  goerli: '0x0000000000000000000000000000000000000000',
+  rinkeby: '0xBf6f1a1C0D494313d0053BDd97714330b6FdAff3',
```
Comment (EndymionJkb) [inline src/mappings/helpers/constants.ts:57]: Are they really deployed on Rinkeby? I don't see any official token among the 7 pages of AAVE's.

**Review by EndymionJkb — CHANGES_REQUESTED** [review]
Is this still needed, given that we don't have  AAVE/USD pool?

File: src/mappings/helpers/constants.ts (hunk 2 above)
Comment (gtaschuk) [reply src/mappings/helpers/constants.ts:57]: Looks like the token was deprecated ~6 months ago https://rinkeby.etherscan.io/address/0xBf6f1a1C0D494313d0053BDd97714330b6FdAff3 hmm

**Comment by gtaschuk** [comment]
> Is this still needed, given that we don't have AAVE/USD pool?

Yes, as discussed in slack, we would need a AAVE/USD pair to get USD pricing data

**Pull request closed without merging**